### PR TITLE
bump Poetry in action and remove requirements.txt

### PR DESCRIPTION
**Pull request by hms5232**
TL;DR

1. Bump Poetry version (1.1.14 -> 1.2.2 & 1.3.1) in action pipeline
2. Remove `requirements.txt`

------

In #105 , I think it is time to update Poetry. Both of 1.2 and 1.3 add many breaking but useful features (ex: lock file format 2.0). However, some device may face issue when they upgrade from 1.1. For example, my personal computer and raspberry pi 4 work fine, but device my job use failed in upgrade.

Because of those, I set two Poetry version matrix to cover 1.2 and 1.3, but drop 1.1. This mean all maintainer of this project should not use Poetry before version 1.2.

Also, I remove the `requirements.txt`, maintain `pyproject.toml` and `poetry.lock` is the only thing need to do of dependencies management.

If everyone ready, let's merge and move one step closer to the goal (release new packae)!

### CODE REVIEW

**Comment by bluet** [comment]
@hms5232 cool thanks 👍
